frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -336,3 +336,71 @@ export const setMineVerifiedStatus = (mine_guid, payload) => (dispatch) => {
     })
     .catch(() => dispatch(error(reducerTypes.SET_MINE_VERIFIED_STATUS)));
 };
+
+// mine subscription
+export const subscribe = (mineGuid) => (dispatch) => {
+  dispatch(request(reducerTypes.SUBSCRIBE));
+  dispatch(showLoading());
+  return axios
+    .post(ENVIRONMENT.apiUrl + API.SUBSCRIPTION(mineGuid), {}, createRequestHeader())
+    .then(() => {
+      notification.success({
+        message: "Successfully subscribed",
+        duration: 10,
+      });
+      dispatch(success(reducerTypes.SUBSCRIBE));
+      dispatch(hideLoading());
+    })
+    .catch((err) => {
```

Comment:
@james-hollinger  Can you recommend how to use the new interceptors here?

Reply:
1. Change line 344 from `return axios` to `return CustomAxios()`
2. remove the `notification.error({...` stuff. that's handled automatically by CustomAxios now.
3. I would recommend moving the hideloading() calls to a finally block at the end to reduce code duplication.

See some of the other ActionCreators in this file for examples